Branch-SDK/Branch-SDK/Networking/BNCNetworkServiceProtocol.h

```diff
@@ -0,0 +1,80 @@
+//
+//  BNCNetworkServiceProtocol.h
+//  Branch-SDK
+//
+//  Created by Edward Smith on 5/30/17.
+//  Copyright © 2017 Branch Metrics. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
```

Comment:
Can you add a comment up here with a high level description of how to implement this. IE:
1. Implement `BNCNetworkServiceProtocol` class, and specifically `new` and `networkOperationWithURLRequest`. Others are optional.
2. Implement the `BNCNetworkOperationProtocol`, and specifically the `start` method. `cancel` is optional.
3. In the `networkOperationWithURLRequest` method, create your class that conforms to the `BNCNetworkOperationProtocol`
4. In your app delegate, call the static method `setCustomNetworkClass` on Branch with your class that conforms to `BNCNetworkServiceProtocol`.